package.json

```
@@ -0,0 +1,106 @@
+{
+  "name": "plugin-paginate-graphql",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "0.0.0-development",
+  "description": "Octokit plugin to paginate GraphQL API endpoint responses",
+  "main": "index.js",
+  "scripts": {
+    "build": "pika-pack build",
+    "lint": "prettier --check '{src,test}/**/*' README.md package.json",
+    "lint:fix": "prettier --write '{src,test}/**/*' README.md package.json",
+    "pretest": "npm run -s lint",
+    "test": "jest --coverage",
+    "test:e2e": "jest --testRegex test/*.e2e.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/octokit/plugin-paginate-graphql.js.git"
+  },
```

We use a short form now

```suggestion
  "repository": "github:octokit/plugin-paginate-graphql.js",",
```